rubyau/constitution.md

```diff
@@ -1,468 +1,779 @@
-  3. in any other case, in the form set out in Appendix 3. 
+1. Subject to subrule 3 and rule 56, a committee member holds office until the positions of the Committee are declared vacant at the next annual general meeting or special general meeting whichever falls on the 12 month anniversary of the member’s appointment.
 
-## 21. Committee of management
+2. A committee member may be re-elected.
 
-1. The affairs of the Association shall be managed by the committee of management. 
+3. A general meeting of the Association may—
+  1. by special resolution remove a committee member from office; and 
+  2. elect an eligible member of the Association to fill the vacant position in accordance with this Division.
 
-2. The committee—
-  1. shall control and manage the business and affairs of the Association; and 
-  2. may, subject to these Rules, the Act and the Regulations, exercise all such  powers and functions as may be exercised by the Association other than those powers and functions that are required by these Rules to be exercised by general meetings of the members of the Association; and 
-  3. subject to these Rules, the Act and the Regulations, has power to perform all such acts and things as appear to the committee to be essential for the proper management of the business and affairs of the Association. 
+4. A member who is the subject of a proposed special resolution under subrule 3(i) may make representations in writing to the Secretary or President of the Association (not exceeding a reasonable length) and may request that the representations be provided to the members of the Association.
 
-3. Subject to section 23 of the Act, the committee shall consist of a minimum of four (4) and a maximum of six (6) members each of whom shall be elected at the annual or special general meetings of the Association, at least once per year. 
+5. The Secretary or the President may give a copy of the representations to each member of the Association or, if they are not so given, the member may require that they be read out at the meeting at which the special resolution is to be proposed.
 
-## 22. Office holders
+#### 55 Vacation of office
 
-1. The officers of the Association shall be—
-  1. a President; 
-  2. a Vice-President; 
-  3. a Treasurer; and 
-  4. a Secretary. 
+1. A committee member may resign from the Committee by written notice addressed to the Committee.
 
-2. The provisions of section ‘Election of officers and ordinary committee members’, so far as they are applicable and with the necessary modifications, apply to and in relation to the election of persons to any of the offices referred to in subrule (1). 
+2. A person ceases to be a committee member if he or she—
+  1. ceases to be a member of the Association; or
+  2. fails to attend 3 consecutive committee meetings (other than special or urgent committee meetings) without leave of absence under rule 67; or
```

Comment:
Rule 67 is referenced here, but the relevant rule appears to be Rule 65 (Leave of absence). Can someone confirm this is correct and fix this reference?